woocommerce/woocommerce
# TT2: Update "View Products" button padding so it's consistent with add to basket button.

### All Submissions:






### Changes proposed in this Pull Request:

As discovered [here](https://github.com/woocommerce/woocommerce/pull/37018#issuecomment-1452153839) there is some inconsistent padding between the "Add to basket" button and the "View products" so when these elements are alongside each other they look misaligned. This PR makes the padding and alignment consistent.

Closes https://github.com/woocommerce/woocommerce/issues/37077



### How to test the changes in this Pull Request:



- [c] Have you followed the [Writing high-quality testing instructions guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions)?

1. Activate TT2 theme.
2. Ensure you have some group products as well as simple products.
3. Go to Shop page.
4. Both "View products" and "Add to basket" buttons should look consistent in terms of spacing and padding.


| Before      | After |
| ----------- | ----------- |
| ![Screenshot 2023-03-06 at 12 53 49](https://user-images.githubusercontent.com/8639742/223118139-94782a04-1e4f-4fad-af22-73706d82a84e.png) | ![Screenshot 2023-03-06 at 12 54 26](https://user-images.githubusercontent.com/8639742/223118169-e983ba29-6d06-46ff-a68a-a0b7516b36e6.png)|



### Other information:




### FOR PR REVIEWER ONLY:
